hackathon_site/review/migrations/0003_reviewergoup.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.1.8 on 2021-06-04 02:42
+
+from django.db import migrations
+from django.core.management.sql import emit_post_migrate_signal
+from review import REVIEWER_PERMISSIONS
```

Comment:
Since this import is here, the migration is not static - changing this variable will cause the migration to have different results for different people. Hard-code the permissions into this file